src/App.js

```diff
@@ -1,13 +1,53 @@
+import {
+  BrowserRouter as Router, Routes, Route, Link,
+} from 'react-router-dom';
 import Calculator from './components/Calculator';
-import './App.css';
 import Quote from './components/quote';
+import Home from './components/Home';
+import './App.css';
+
+function Header() {
+  return (
+    <header>
+      <h1>Math Magicians</h1>
+      <h2>Welcome to our page!</h2>
+    </header>
+  );
+}
 
 function App() {
   return (
-    <div>
-      <Calculator />
-      <Quote />
-    </div>
+    <Router>
+      <div className="firstcontainer">
+        <div className="navbar">
+          <nav>
+            <ul>
+              <li>
+                <Link to="/">Home</Link>
+              </li>
+              <li>
+                <Link to="/calculator">Calculator</Link>
+              </li>
+              <li>
+                <Link to="/quote">Quote</Link>
+              </li>
+            </ul>
+          </nav>
+        </div>
```

Comment:
- [ ] you know you can actually make this into a component and import it here right? :+1: